### DISCUSSION TITLE
S3 bucket - for tf state

### DISCUSSION THREAD

**Discussion by mirik12** [GeekOpsUA/aws_eks_karpenter_mf#6] [General]
Поділіться своїми ідеями як зберігати TF state? Можливо нам потрібен модуль для цього чи  буде описана в env? Сергій пропонує обговорити зберігання в S3 бакеті

**Comment by oleksiihead** [comment]
S3 + DynamoDB - це розповсюджений кейс
https://developer.hashicorp.com/terraform/language/settings/backends/s3
